tests/loggers/test_base.py

```diff
@@ -176,6 +176,76 @@ def training_step(self, batch, batch_idx):
     assert logger2.finalized_status == "success"
 
 
+def test_trainer_loggers_property():
```

Comment:
since these tests are trainer property specific, should we move them to `tests/trainer/properties/test_logger.py`?

Reply:
Will do. I'll call it `test_loggers.py` though, since trainer.loggers will be the preferred option going forward.